.travis.yml

```diff
@@ -6,7 +6,15 @@ rvm:
   - 2.4.4
   - 2.5.1
 
-script: bundle exec rspec
+install:
+  - bundle install --jobs=3 --retry=3
+  - |-
```

Comment:
why is this needed?

Reply:
Following the security team's guidance: https://github.com/mdsol/fossa_ci_scripts/blob/master/travis_ci/SETUP.md

Reply:
Sorry for not getting back to this.  I don't seem to get notifications on GitHub.

Travis treats the above statement's ':' character as a section delimiter and garbles the rest of the command.  I've tried wrapping the entire line in double quotes, but using the yml literal operator (|-) has been the only way to consistently get this command to execute.  It's important to note that due to either Travis or yml funkyness all the characters have to line up just right too, with the 'c' from 'curl' under the '|' from '|-'.